src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -30,9 +31,15 @@ private static void setupPresetTests(Application application) {
 				DriverFeature.getDriverManager());
 		SelectSecondTestFigureOptionListener selectSecondTestFigureOptionListener = new SelectSecondTestFigureOptionListener(
 				DriverFeature.getDriverManager());
+		SelectFigureOptionListener selectTriangleOptionListener = new SelectFigureOptionListener(DriverFeature.getDriverManager(), "Triangle");
+		SelectFigureOptionListener selectRectangleOptionListener = new SelectFigureOptionListener(DriverFeature.getDriverManager(), "Rectangle");
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.